docs/sources/installation/docker.md

```diff
@@ -137,6 +137,10 @@ Replace `Dockerfile` in above example with `ubuntu.Dockerfile` to build a custom
 
 This section contains important information if you want to migrate from previous Grafana container versions to a more current one.
 
+### Migrate to v7.3 or later
+
+Grafana Docker image run with the `root` group (id 1) instead of the `grafana` group (id 472) for better compatibility with OpenShift. If you extend the official docker image you may need to change your scripts to use the `root` group instead of the `grafana`.
```

Comment:
```suggestion
The Grafana Docker image runs with the `root` group (id 1) instead of the `grafana` group (id 472), for better compatibility with OpenShift. If you extend the official Docker image you may need to change your scripts to use the `root` group instead of `grafana`.
```